config/pipelines/bespoke_pcr.yml

```diff
@@ -36,3 +46,59 @@ Bespoke PCR MX:
   relationships:
     LBB Lib PCR-XP: LBB Lib Pool Stock
     LBB Lib Pool Stock: LB Lib Pool Norm
+Bespoke PCR Enriched BCR:
+  filters:
+    request_type_key: limber_pcr_bespoke
+    library_type:
+      - Manual Chromium single cell BCR
+  library_pass: LBB Lib PCR-XP
+  relationships:
+    LBB Ligation: LBB Enriched BCR
+    LBB Enriched BCR: LBB Lib PCR-XP
+Bespoke PCR Enriched BCR HT:
+  filters:
+    request_type_key: limber_pcr_bespoke
+    library_type:
+      - Manual Chromium single cell BCR HT
+  library_pass: LBB Lib PCR-XP
+  relationships:
+    LBB Ligation: LBB Enriched BCR HT
+    LBB Enriched BCR HT: LBB Lib PCR-XP
+Bespoke PCR Enriched TCR:
+  filters:
+    request_type_key: limber_pcr_bespoke
+    library_type:
+      - Manual Chromium single cell TCR
+  library_pass: LBB Lib PCR-XP
+  relationships:
+    LBB Ligation: LBB Enriched TCR
+    LBB Enriched TCR: LBB Lib PCR-XP
+Bespoke PCR Enriched TCR HT:
+  filters:
+    request_type_key: limber_pcr_bespoke
+    library_type:
+      - Manual Chromium single cell TCR HT
+  library_pass: LBB Lib PCR-XP
+  relationships:
+    LBB Ligation: LBB Enriched TCR HT
+    LBB Enriched TCR HT: LBB Lib PCR-XP
+Bespoke PCR Chromium Alternative Entry:
+  filters:
+    request_type_key: limber_pcr_bespoke
+    library_type:
+      - Chromium single cell HTO
+      - Chromium single cell surface protein
+      - Chromium single cell surface protein HT
+      - Chromium Visium
+      - Chromium Visium FFPE
+      - Manual Chromium single cell 3 prime v3
+      - Manual Chromium single cell 3 prime HT v3
+      - Manual Chromium single cell 5 prime v2
+      - Manual Chromium single cell 5 prime HT v2
+      - Manual Chromium single cell TCR
+      - Manual Chromium single cell BCR
+      - Manual Chromium single cell BCR HT
+      - Manual Chromium single cell TCR HT
```

Comment:
Not sure if this is the correct selection of library types, or if I should restrict it further.